benchmark/websocket/simple.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
```

Comment:
if this was taken from ws, you probably need to add ws' license here.

Reply:
I removed the 'ws' package.